functionaltests/internal/esclient/client.go

```diff
@@ -39,16 +39,15 @@ type Client struct {
 	es *elasticsearch.TypedClient
 }
 
-// New returns a new Client for querying APM data.
-func New(cfg Config) (*Client, error) {
+// New returns a new Client for accessing Elasticsearch.
+func New(esURL, username, password string) (*Client, error) {
 	transport := http.DefaultTransport.(*http.Transport).Clone()
-	transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: cfg.TLSSkipVerify}
+	transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: false}
```

Comment:
Do we need to keep the cloned transport if we don't allow modifying this value? I'm not sure if there is a use case for configuring `InsecureSkipVerify`. If there isn't I would remove this.